tests/samplers_tests/test_qmc.py

```diff
@@ -240,29 +240,26 @@ def test_sample_relative_sobol() -> None:
         ]
     )
 
-    # If empty search_space, return {}
+    # If empty search_space, return {}.
     np.testing.assert_allclose(samples, ref_samples, rtol=1e-6)
 
 
+@pytest.mark.parametrize("seed", [0, 12345])
 @pytest.mark.parametrize("scramble", [True, False])
 @pytest.mark.parametrize("qmc_type", ["sobol", "halton"])
-def test_sample_relative_seeding(scramble: bool, qmc_type: str) -> None:
+def test_sample_relative_seeding(seed: int, scramble: bool, qmc_type: str) -> None:
```

Comment:
(This comment is optional.)
I guess this code follows my suggestion, but it may be more natural if the order of arguments is consistent with `_init_QMCSampler_without_exp_warning`.

How about reordering them like `scramble`, `qmc_type` and `seed`?

Sorry for bothering you.

Reply:
No worries. I've fixed it too.

Reply:
Thank you!